src/app/modules/dashboard/Dashboard.jsx

```diff
@@ -1,7 +1,8 @@
-import { Component } from 'react'
+import React from 'react'
```

Comment:
seems you don't use `React` in this component. Why not just import `Component` from `React` ?